src/lib/tls/asio/asio_async_ops.h

```diff
@@ -12,7 +12,7 @@
 #include <botan/types.h>
 
 #include <boost/version.hpp>
-#if BOOST_VERSION >= 106600
+#if BOOST_VERSION >= 107300
```

Comment:
My main concern here is this increases the required Boost version (which is fine) but also it's hard for downstream users to know if the asio integration works for their botan+boost combination. They can test if it's available, using the usual feature macro - but they don't know if it will work if they include the header.

Maybe we should define somewhere (in an existing or new header) what our minimum boost version is, wrt this asio integration, which would allow applications to check for availability. (Or maybe you have some better idea of how to address this)

Reply:
That's indeed a valid point. I'll make a proposal in a follow-up PR.